src/app/globals.css

```diff
@@ -2,6 +2,7 @@
   --primary-font: white;
   --red-font: #ea1f1f;
   --blue-font: #292fe0;
+  --black-font: black;
```

Comment:
stick with hex codes in general instead of mixing both for consistency. so #000000 here